django_1/book_app/api/views.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.viewsets import ModelViewSet
+from book_app.models import Book
+from book_app.api.serializers import BookSerializer
+
+
+
+class BookModelViewSet(ModelViewSet):
+    queryset = Book.objects.all()
+    serializer_class = BookSerializer
+    filterset_fields = ('__all__')
```

Comment:
nit: better to put all field names explicitly to see what actually is going to be used